e2e/app/page/workspace-edit-page.ts

```diff
@@ -464,4 +464,12 @@ export default class WorkspaceEditPage extends WorkspaceBase {
     const element = Checkbox.findByName(this.page, FIELD.shareWithCollaboratorsCheckbox.textOption);
     await element.check();
   }
+
+  // Fill out only the fields needed for a successful duplication
+  async fillOutRequiredDuplicationFields(): Promise<string> {
```

Comment:
👍🏻

Reply:
The refactoring idea I mentioned earlier is to use this in more places.

Not quite ready PR: https://github.com/all-of-us/workbench/pull/4877